src/components/label-pill/LabelPill.tsx

```diff
@@ -0,0 +1,43 @@
+import * as React from 'react';
+import classNames from 'classnames';
+import LabelPillIcon from './LabelPillIcon';
+import LabelPillText from './LabelPillText';
+
+import './LabelPill.scss';
+
+export enum LabelPillStatus {
+    DEFAULT = 'default',
+    INFO = 'info',
+    HIGHLIGHT = 'highlight',
+    SUCCESS = 'success',
+    WARNING = 'warning',
+    ALERT = 'alert',
+    ERROR = 'error',
+}
+
+export interface LabelPillProps {
+    children: React.ReactNode;
```

Comment:
React.ReactChild

Reply:
This can have a single of multiple instances of its subcomponents, so using `React.ReactChild | Array<React.ReactChild>`